ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/History/EraParams.hs

```diff
@@ -219,11 +218,11 @@ instance Serialise SafeZone where
       (1, 1) -> return UnsafeIndefiniteSafeZone
       _ -> fail $ "SafeZone: invalid size and tag " <> show (size, tag)
 
--- | Artificial encoder for backward compatibility, see #2646.
+-- | Artificial encoder for backward compatibility, see ouroboros-network#2646.
 encodeSafeBeforeEpoch :: Encoding
 encodeSafeBeforeEpoch = encodeListLen 1 <> encodeWord8 0
 
--- | Artificial decoder for backward compatibility, see #2646.
+-- | Artificial decoder for backward compatibility, see ouroboros-network#2646.
```

Comment:
I guess a low-prio thing to do would be to actually fully remove this artifact, in a similar way as we did for when `GenesisWindow` was added to `EraParams` in 3fc6d28f00b0a1b0d3187cb7916f1e192dd79fcb (with the compat code removed in e477ed0e5dc5d9cbdac60e8824a7bda411d550b4).